lib/consult/template_functions.rb

```diff
@@ -21,7 +21,7 @@ def service(key, scope: :all, options: {}, meta: {})
     end
 
     # Execute a prepared query
-    def query(name_or_id, options: nil)
+    def query(name_or_id, options: {})
```

Comment:
I modified this to keep this variable options consistent as the other Consul methods (service and key), they are also having as default an empty hash